weaver/sdks/fabric/interoperation-node-sdk/protos-js/common/access_control_pb.d.ts

```diff
@@ -0,0 +1,61 @@
+// package: common.access_control
```

Comment:
We should not commit this `protos-js`, its a duplicate of the one in `weaver/common/protos-js`.
While building `sdk` using `make build-local`, the makefile will copy it here...

Reply:
I agree. Having duplicates carries the risk of inconsistency between the sdk and the protos. If we are doing a local build, it's easy to rebuild the protos (if required) and copy here. For the regular build, there's a dependency to the protos package published to NPM anyway, so there's no need to fetch the built JS code.